client/state/purchases/selectors/will-atomic-site-revert-after-purchase-deactivation.js

```diff
@@ -22,8 +24,13 @@ export const willAtomicSiteRevertAfterPurchaseDeactivation = ( state, purchaseId
 
 	const purchase = getByPurchaseId( state, purchaseId );
 
-	const isAtomicSupportedProduct = ( productSlug ) =>
-		planHasFeature( productSlug, WPCOM_FEATURES_ATOMIC );
+	const isAtomicSupportedProduct = ( productSlug ) => {
+		if ( isEnabled( 'marketplace-starter-plan' ) && isMarketplaceProduct( state, productSlug ) ) {
+			return true;
+		}
+
+		return planHasFeature( productSlug, WPCOM_FEATURES_ATOMIC );
```

Comment:
I wish `planHasFeature( productSlug, WPCOM_FEATURES_ATOMIC )` would also work for marketplace products, so I don't need the hardcoded plan/product check above, but I think that's not trivial because the marketplace products are not listed in https://github.com/Automattic/wp-calypso/blob/trunk/packages/calypso-products/src/plans-list.tsx